### ISSUE TITLE
Multi-drop device communication using OSDP

### ISSUE THREAD

**Issue by DeepaGRao** [bytedreamer/OSDP.Net#87]
Hi,
In the previous query I had shared the sample code and below are details in response to your query.
Sorry for being late in my response...

We are using multiple devices multi-dropped and connected over RS-485. 
Serial to Ethernet converter that we are using is MOXA NPort 6650-16
With the latest 2.1.6, I am still facing the issue. If among the 3 multi-dropped devices, one of the device is switched off, we are not able to get response from any of the other devices
Can you please help on this?

**Comment by bytedreamer** [comment]
I did everything that can be done to get it to work properly with the USR-TCP232-410s RS-485 converter. What I found is that TCP connection would randomly disconnect when a device is switched off. Logic has been added to reconnect without affecting the other devices. There is not much I can do without a more direct engagement.

Instead of connecting via TCP, it would be worth trying the virtual port driver that emulates a serial port. https://www.moxa.com/en/support/product-support/software-and-documentation/search?psid=50388

Contact my company at https://z-bitco.com for further support.
